core/logger/app-logger.js

```diff
@@ -2,6 +2,14 @@ import * as winston from 'winston'
 import * as rotate from 'winston-daily-rotate-file'
 import config from '../config/config.dev'
 
+const fs = require('fs');
```

Comment:
Excellent! Can you please change the require statement to be es6 style import? Same as line 3 :-)